src/meta/assessment/originalDataPoint/originalDataPoint.ts

```diff
@@ -22,15 +22,16 @@ export interface OriginalDataPointValues {
 }
 
 export interface OriginalDataPoint {
-  id: number
+  commentsExtentOfForest?: string
+  commentsForestCharacteristics?: string
```

Comment:
Would it be better to have a composite field : 

comments: { extentOfForest, forestCharacteristics }

Reply:
I would also propose to pass around the type TableNames (to not hardcode it )